build.sh

```diff
@@ -81,10 +85,23 @@ declare -a arr=(master
 
 for version in "${arr[@]}"
 do
+    recreate=false
     schema=https://raw.githubusercontent.com/kubernetes/kubernetes/${version}/api/openapi-spec/swagger.json
     prefix=https://raw.githubusercontent.com/${REPO}/master/${version}/_definitions.json
 
-    openapi2jsonschema -o "${version}-standalone" --kubernetes --stand-alone "${schema}"
-    openapi2jsonschema -o "${version}-local" --kubernetes "${schema}"
-    openapi2jsonschema -o "${version}" --kubernetes --prefix "${prefix}" "${schema}"
+    if [ "${recreate}" = true ] || [ ! -d "${version}-standalone-strict" ]; then
+        openapi2jsonschema -o "${version}-standalone-strict" --kubernetes --stand-alone --strict "${schema}"
+    fi
+
+    if [ "${recreate}" = true ] || [ ! -d "${version}-standalone" ]; then
+        openapi2jsonschema -o "${version}-standalone" --kubernetes --stand-alone "${schema}"
+    fi
+
+    if [ "${recreate}" = true ] || [ ! -d -o "${version}-local" ]; then
```

Comment:
There is an extra `-o` here that is incorrect.

Reply:
Oh woops - I tested what would happen with this this mistake of including `-o` and realized that this means that whatever I ran with `recreate=false` with a non-existing `...-local` directory were never created.

I'll make sure to fix this, and generate potentially missing `...-local` directories.

---

```
# the empirical test I made to determine what adding -o caused
mkdir test
if [ ! -d -o "test" ]; then echo "1"; fi
if [ ! -d "test" ]; then echo "2"; fi
if [ ! -d -o "testNoExistance" ]; then echo "3"; fi
if [ ! -d "testNoExistance" ]; then echo "4"; fi
# output was "4" only
```